BraveShared/Analytics/UserReferralProgram.swift

```diff
@@ -47,6 +47,7 @@ public class UserReferralProgram {
     
     /// Looks for referral and returns its landing page if possible.
     public func referralLookup(refCode: String?, completion: @escaping (_ refCode: String?, _ offerUrl: String?) -> Void) {
+        Preferences.URP.referralLookupOutstanding.value = false
```

Comment:
Doesn't make sense to me, shouldn't this flag be toggled after network call?

Reply:
Yup 😄 I think I moved this last minute when cleaning things up, but this looks v wrong. Fixing.